climada/engine/impact.py

```diff
@@ -148,8 +148,7 @@ def calc(self, exposures, impact_funcs, hazard, save_mat=False):
         --------
             Use Entity class:
 
-            >>> haz = Hazard('TC') # Set hazard
-            >>> haz.read_mat(HAZ_DEMO_MAT)
+            >>> haz = Hazard.from_mat(HAZ_DEMO_MAT)  # Set hazard
```

Comment:
Should it not read
```suggestion
            >>> haz = Hazard.from_mat(HAZ_DEMO_MAT, haz_type='TC')  # Set hazard
```
?

Reply:
Not in this case - the `from_mat` method overwrites the tag with whatever it finds in the MATLAB file